### DISCUSSION TITLE
What research papers are used for the Deep Doc ocr ? and table extraction if any ?

### DISCUSSION THREAD

**Discussion by Siddhu2502** [infiniflow/ragflow#1573] [Q&A]
Just wanted to know if there were any research papers that are being used for the DEEPDOC part

**Comment by yingfeng** [comment]
Hi, currently, 
DeepDoc is trained using YOLO, as a result, it does not have any research papers. 
Previously, we've tried [table transformer](https://github.com/microsoft/table-transformer/), [ppstructure](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/ppstructure/README.md), the effects are not satisfying. These works have publications and you could find it in their README.
